src/src/Controller/MediaController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class MediaController extends AbstractController
+{
+    #[Route('/media', name: 'app_media')]
+    public function index(): Response
+    {
+        return $this->render('media/index.html.twig', [
+            'controller_name' => 'MediaController',
```

Comment:
This variable is an example generated by the framework and you should remove it since you don't use it in the twig template.